sdk/cosmos/azure-cosmos-spark_3_2-12/src/main/scala/com/azure/cosmos/spark/CosmosConfig.scala

```diff
@@ -86,6 +86,7 @@ private[spark] object CosmosConfigNames {
   val WritePatchDefaultOperationType = "spark.cosmos.write.patch.defaultOperationType"
   val WritePatchColumnConfigs = "spark.cosmos.write.patch.columnConfigs"
   val WritePatchFilterPredicate = "spark.cosmos.write.patch.filter"
+  val WritePatchBulkUpdateColumnConfigs = "spark.cosmos.write.patchBulkUpdate.columnConfigs"
```

Comment:
Please remove the patch prefix here - we should call the feature publicly "BulkUpdate" - not patchBulUpdate - otherwise the whole discussion why not all patch operations are possibel etc. will come up

Reply:
make sense, updated the public area to ItemBulkUpdate (The internal methods still prefixed with patch)